Analytics/manage.py

```diff
@@ -9,17 +11,20 @@
 from create_celery import make_celery
 from add_startup_admin import AddStartupAdmin
 
+
 application = create_app()
 celery_task = make_celery(application)
 manager = Manager(app=application)
 manager.add_command('db', MigrateCommand)
 
 manager.add_command('runserver', Server(
-    use_debugger=True,
+    use_debugger=False,
     use_reloader=True,
-    host='0.0.0.0'
+    host='api'
```

Comment:
The line `host='api'` is only applicable to the docker specific way of working, but not the ubuntu deployment.

Please specify this in the deployment doc for both deployment methods and identify as a conditional value, perhaps using `<host-value>`.